pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -87,6 +87,11 @@ func init() {
 	cloudprovider.RegisterCloudProvider(ProviderName, func(config io.Reader) (cloudprovider.Interface, error) { return newGCECloud(config) })
 }
 
+// Raw access to the underlying GCE service, probably should only be used for e2e tests
```

Comment:
I guess this is probably fine, though leaking the interface still feels kinda dirty (and I worry that folks might abuse this)? it almost seems like `ListForwardingRules` should have been included in #16142. @bprashanth